packages/components/.build/stencil/stencil.bindings.angular.ts

```diff
@@ -78,13 +78,6 @@ export const AngularGenerator = () =>
           'bal-hint-text',
         ],
       },
-      'bal-stage': {
```

Comment:
why did you remove the stage ?

Reply:
its duplicate, intelij reported :), look at line 66 :)

Reply:
okay nice thanks